.github/codespell-ignorelist

```diff
@@ -0,0 +1 @@
+crate
```

Comment:
https://github.com/codespell-project/codespell/issues/2281

This wont be fixed from what it seems so this will need to be here.